install.sh

```diff
@@ -54,10 +54,9 @@ install_dvm_helper() {
   [ $(getconf LONG_BIT) == 64 ] && DVM_ARCH="amd64" || DVM_ARCH="386"
 
   # Download latest release
-  LATEST_TAG=$(curl -s https://api.github.com/repos/getcarina/dvm/tags | grep name -m 1 | awk '{print $2}' | cut -d '"' -f2)
   mkdir -p "$DVM_DIR/dvm-helper"
   bin="$DVM_DIR/dvm-helper/dvm-helper"
-  url=https://github.com/getcarina/dvm/releases/download/$LATEST_TAG/dvm-helper-$DVM_OS-$DVM_ARCH
+  url=https://download.getcarina.com/dvm/latest/$DVM_OS/$DVM_ARCH/dvm-helper
```

Comment:
These need adjustment to be using `uname -s` for DVM_OS and `uname -m` for DVM_ARCH (replacing the `getconf` bits above).

Sample run that works for me:

```
$ curl https://download.getcarina.com/dvm/latest/$(uname -s)/$(uname -m)/dvm-helper -o dvmhelpah; chmod u+x dvmhelpah; ./dvmhelpah --version
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 7160k  100 7160k    0     0   293k      0  0:00:24  0:00:24 --:--:--  440k
Docker Version Manager version 0.1.4 (ca12e09)
```